cfg/2.0.0/definitions.yaml

```diff
@@ -2792,13 +2792,20 @@ groups:
                 
                 zypper remove xorg-x11*
       - check:
-          audit: "apt-cache show xserver-xorg*"
+          audit: "apt-cache policy xserver-xorg* | grep Installed -B1"
+          use_multiple_values: true
           constraints:          
             platform:
             - ubuntu18
-          tests:          
+          tests:
+            bin_op: or
```

Comment:
Because we also take the line before "Installed: (none)" - aka the package name we check if Installed present and then check its (none) or it isn't present at all then its the package name which is OK